src/lib/server/database.ts

```diff
@@ -90,3 +91,10 @@ export async function createLabel(title: Label['title'], color: Label['color'])
     strictEqual(rest.length, 0);
     return LabelSchema.pick({ label_id: true }).parse(first);
 }
+
+/** Creates a new {@linkcode Dept} or department. Requires only the department name as input.  */
+export async function createDept(name: Dept['name']) {
+    const [first, ...rest] = await sql`SELECT create_dept(${name}) AS dept_id`.execute();
+    strictEqual(rest.length, 0);
+    return DeptSchema.pick({ dept_id: true }).parse(first);
```

Comment:
Don't forget to extract the property.
```suggestion
    return DeptSchema.pick({ dept_id: true }).parse(first).dept_id;
```